core/src/main/scala/com/eharmony/spotz/optimizer/grid/GridSearchResult.scala

```diff
@@ -17,15 +17,7 @@ class GridSearchResult[P, L](
   extends OptimizerResult[P, L](bestPoint, bestLoss) {
 
   override def toString = {
```

Comment:
Do you want to make another function call here as `toString` is already overridden for `case classes`?